Sources/Testing/Testing.docc/MigratingFromXCTest.md

```diff
@@ -200,7 +200,7 @@ the presence of the `@Test` attribute:
 }
 
 As with XCTest, the testing library allows test functions to be marked `async`
-and/or `throws` and to be isolated to a global actor (for example, by using the
+or `throws`, or `async`-`throws`, and to be isolated to a global actor (for example, by using the
```

Comment:
Did you mean "to be marked `async`, `throws`, or `async throws`"?

Reply:
Yep. I was following the style we use in TSPL for Concurrency.

Reply:
Hmm. Alright, if that's the style used there. Let's make sure the commas make sense though.

Reply:
```suggestion
`throws`, or `async`-`throws`, and to be isolated to a global actor (for example, by using the
```